docs/docs-content/vm-management/create-manage-vm/advanced-topics/migrate-vm-kubevirt.md

```diff
@@ -119,26 +177,29 @@ from VMware vSphere to Palette VMO.
+   | **Migration Namespace**                                            | Namespace where the migration VM is created. The namespace must exist on the cluster. You can enter the namespace you created earlier or use the `default` namespace.                                                     |                                              |
+   | **vSphere Endpoint**                                               | Your vSphere endpoint. You can specify a Full Qualified Domain Name (FQDN) or an IP address. Make sure you specify the endpoint without the HTTP scheme `https://` or `http://`. For example, `vcenter.mycompany.com`.    |                                              |
+   | **vSphere Username (with domain)**                                 | Your vSphere account username.                                                                                                                                                                                            |                                              |
+   | **vSphere Password**                                               | Your vSphere account password.                                                                                                                                                                                            |                                              |
+   | **Allow Insecure Connection (Bypass x509 Verification)**           | Enabling this option bypasses x509 CA verification. In production environments, enter `N` if using a custom registry with self-signed SSL certificates. Otherwise, enter `Y`.                                             | `Y`/`n`                                      |
+   | **Use VDDK (recommended)?**                                        | Specify whether to provide a VMware Virtual Disk Development Kit (VDDK) image.                                                                                                                                            | `Y` / `n`                                    |
+   | **VDDK image**                                                     | Provide the registry URL to the VDDK image. Make sure you specify the registry URL without the HTTP scheme `https://` or `http://`. For example, `docker.io/myorganization/vddk:v8.0.3`.                                  |                                              |
+   | **VDDK image pull secret file path (optional, hit enter to skip)** | If your VDDK image is hosted on a private registry, provide the local filesystem path to your image pull secret file. For example, `~/path/to/image-pull-secret.yaml`.                                                    |                                              |
```

Comment:
These are the three new parameters that will be asked. The remaining parameters are the same.